src/TwabRewards.sol

```diff
@@ -1,31 +1,89 @@
-// SPDX-License-Identifier: GPL-3.0
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.19;
```

Comment:
I think it's good to follow the standards we're using in the other contracts for v5: https://github.com/GenerationSoftware/pt-v5-vault/blob/27aa886482bed5f02856371ccdb1db20901937ef/src/Vault.sol#L2

Is there a downside to not fixing it?

Reply:
Usually it's preferred to avoid compiling the contract with a higher minor version of solc.
For example, you could set solc to version 0.8.21 and the contract would compile.
So at least with a fixed version, you are sure that the contract has been compiled with that version and not a higher one.
You also avoid any bug that may arise when using different Solidity versions across various contracts.

Reply:
We have many other places in the codebase where the exact solidity compiler is specified, but since this is not harmful and I'm not strongly against it, I'm fine with using exact versions in the .sol files.

Changed here: [40eea17](https://github.com/GenerationSoftware/pt-v5-twab-rewards/pull/1/commits/40eea179ac75c51fb8cd6693ef4600ec073a8df6)